Integrations/ProofpointTAP_v2/ProofpointTAP_v2.py

```diff
@@ -18,15 +18,50 @@
 BLOCKED_MESSAGES = "Blocked Messages"
 DELIVERED_MESSAGES = "Delivered Messages"
 
-DATE_FORMAT = "%Y-%m-%dT%H:%M:%S.000Z"
+DATE_FORMAT = "%Y-%m-%dT%H:%M:%SZ"
+
+""" Helper functions """
+
+
+def get_now():
+    """ A wrapper function of datetime.now
+    helps handle tests
+
+    Returns:
+        datetime: time right now
+    """
+    return datetime.now()
+
+
+def get_fetch_times(last_fetch):
+    """ Get list of every hour since last_fetch
+    Args:
+        last_fetch (datetime or str): last_fetch time
+
+    Returns:
+        List[str]: list of str represents every hour since last_fetch
+    """
+    now = get_now() - timedelta(seconds=30)
```

Comment:
why you reduce 30 seconds?

Reply:
Proof Point unable to pull event the from the last 30 seconds so I'm adding it to the time gap.